### PR TITLE
[Feature-docker][all-service] refactor the Dockerfile for smaller size

### PR DESCRIPTION
**Pull request by rockxsj**
## *Tips*
- *Thanks very much for contributing to Apache DolphinScheduler.*
- *Please review https://dolphinscheduler.apache.org/en-us/community/index.html before opening a pull request.*

## What is the purpose of the pull request

``` log
* run in docker only print logs to stdout
* refactor the Dockerfile and build script
* ignore some files and directory
```

## Brief change log

``` log
 .dockerignore                                                                          |  29 ++++++++++++++++++++++++++
 .gitignore                                                                             |   4 +++-
 docker/build/Dockerfile                                                                | 107 +++++++++++++++++++++++++++++++++-------------------------------------------------------------
 docker/build/conf/dolphinscheduler/logback/logback-alert.xml                           |  16 --------------
 docker/build/conf/dolphinscheduler/logback/logback-api.xml                             |  21 -------------------
 docker/build/conf/dolphinscheduler/logback/logback-master.xml                          |  46 -----------------------------------------
 docker/build/conf/dolphinscheduler/logback/logback-worker.xml                          |  20 ------------------
 docker/build/hooks/build                                                               |  12 ++---------
 docker/build/startup.sh                                                                | 118 +++++++++++++++++++++++++++++++++++++++++++++++++++-----------------------------------------------------
 docker/kubernetes/dolphinscheduler/templates/deployment-dolphinscheduler-frontend.yaml |  25 ++++++++++++++++++++++
 script/dolphinscheduler-daemon.sh                                                      |  96 +++++++++++++++++++++++++++++++++++++++++++++---------------------------------------
```

## Verify this pull request

*(Please pick either of the following options)*

This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: docker/build/Dockerfile [hunk 1]
```diff
@@ -1,82 +1,49 @@
-#
```
Comment (khadgarmage) [inline docker/build/Dockerfile:1]: why delete the license header?

File: docker/build/hooks/build [hunk 2]
```diff
@@ -38,16 +38,8 @@ echo "Repo: $DOCKER_REPO"
...
-mvn -B clean compile package -Prelease -Dmaven.test.skip=true
-
-# mv dolphinscheduler-bin.tar.gz file to docker/build directory
-echo -e "mv $(pwd)/dolphinscheduler-dist/target/apache-dolphinscheduler-incubating-${VERSION}-dolphinscheduler-bin.tar.gz $(pwd)/docker/build/\n"
-mv $(pwd)/dolphinscheduler-dist/target/apache-dolphinscheduler-incubating-${VERSION}-dolphinscheduler-bin.tar.gz $(pwd)/docker/build/
-
 # docker build
-echo -e "docker build --build-arg VERSION=${VERSION} -t $DOCKER_REPO:${VERSION} $(pwd)/docker/build/\n"
-sudo docker build --build-arg VERSION=${VERSION} -t $DOCKER_REPO:${VERSION} $(pwd)/docker/build/
+echo -e "docker build -f docker/build/Dockerfile -t $DOCKER_REPO:${VERSION} $(pwd)\n"
```
Comment (khadgarmage) [inline docker/build/hooks/build:42]: alter the path?

File: docker/build/Dockerfile (hunk 1 above)
Comment (rockxsj) [reply docker/build/Dockerfile]: Added back

File: docker/build/hooks/build (hunk 2 above)
Comment (rockxsj) [reply docker/build/hooks/build:42]: Because of at the builder phase, copy src should from root path.

**Comment by CalvinKirs** [comment]
Hi, please resolve the conflict.. If there is no update after 7 days, the PR will be closed, and you can reopen it when your problem is resolved.

**Pull request closed without merging**